packages/frontend/src/translations/en.global.json

```diff
@@ -1678,6 +1678,7 @@
             "desc": "Keep in mind that transactions won't have to be confirmed with 2FA once it's disabled.",
             "disable": "Disable 2FA",
             "keep": "No, keep 2FA",
+            "keyConversionRequired": "<br />Due to the number of 2FA access keys on this account, some access keys will need to be converted to full access keys before 2FA can be disabled. The converted keys can sign transactions without a 2FA prompt, so it is important to complete the 2FA disable process once begun. You will receive one or more additional 2FA prompts to perform this key conversion when disabling 2FA.",
```

Comment:
Just small changes here, adding `<br><br>` in front of `You will receive one or more ...` and warp the last sentence with `<b></b>` maybe to elaborate a bit more?

Some users may not understand the details of this, but it is important to emphasize last sentence a bit more because they are likely to get multiple emails during the process.

Reply:
That's much better, thanks @hcho112 !

This text could probably use some touching up, this was just some placeholder text I put down. Basically we want to convey:
- You have a lot of keys.
- You need to approve more than one 2FA transaction in order to disable 2FA.
- Once you've approved your first transaction, there are now 50 FAKs that can sign transactions without a 2FA prompt (this only applies to transactions outside of `wallet.near.org` as wallet will continue to use the multisig LAK until 2FA is disabled. If you abandon the process after you start, then you've circumvented the security guarantees made by 2FA/multisig. For users exclusively on `wallet.near.org` this incomplete access key state doesn't really weaken security on the account, it just breaks some assumptions.